themes/iceberg.color-theme.json

```diff
@@ -138,7 +136,7 @@
 		"sideBar.background": "#161821",
 		"sideBar.border": "#0f1117",
 		"sideBar.dropBackground": "#84a0c620",
-		"sideBar.foreground": "#6b7089",
+		"sideBar.foreground": "#c6c8d1E6",
```

Comment:
After consideration, I think opaque foreground color is better. What do you think?

Reply:
I made it a little transparent to not make it as light, compared to the editor foreground color (see your comment about focus). Since we're focusing on readability, I agree, it does look better.

Reply:
After hearing your opinion (about Zen Mode and so on), I have changed my thought. If users want to focus a content, they can use Zen Mode or hide sidebar. In this case we should focus on readability I think.